src/CollapsedForwarding.cc

```diff
@@ -146,6 +151,16 @@ CollapsedForwarding::HandleNotification(const Ipc::TypedMsgHdr &msg)
     HandleNewData("after notification");
 }
 
+/// Handle messages in IPC queue without relying on the queue state being in
+/// sync with our state (because the queue may be reflecting the state of our
+/// killed predecessor). See also: IpcIoFile::HandleMessagesAtStart().
```

Comment:
```suggestion
/// killed predecessor).
/// \sa IpcIoFile::HandleMessagesAtStart()
```

Reply:
Done in commit 75fa9f5874ba840998a76c43530d0e9571db46c8.